media-libs/libsoundtouch/libsoundtouch-2.1.0.recipe

```diff
@@ -18,16 +18,23 @@ SOURCE_URI="https://gitlab.com/soundtouch/soundtouch/-/archive/$portVersion/soun
 CHECKSUM_SHA256="3fb740c7ac4b304342fbf11a91d14fa3e296269b9fec6266609c4620477d0ad6"
 SOURCE_DIR="soundtouch-$portVersion"
 
+commandSuffix=$secondaryArchSuffix
+commandBinDir="$binDir"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir="$prefix"/bin
+fi
+
 libVersion="1.0.0"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 portVersionCompat="$portVersion compat >= 1"
 
-ARCHITECTURES="!x86_gcc2 x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
```diff
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
 
 commandSuffix=$secondaryArchSuffix
-commandBinDir="$binDir"
+commandBinDir=$binDir
 if [ "$targetArchitecture" = x86_gcc2 ]; then
 	commandSuffix=
-	commandBinDir="$prefix"/bin
+	commandBinDir=$prefix/bin
 fi
+
 libVersion="1.0.0"
 libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 portVersionCompat="$portVersion compat >= 1"
- 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86"

 PROVIDES="
```
* [Using the commandBinDir User Variable](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable)
* [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Reply:
Remark: If I'm not mistaken, most recipes which define commandBinDir don't use quotes in their definition of `commandBinDir`.
This was true for **all** recipes until recently. I don't know why @korli decided to add quotes there. They are completely unneeded.
And even if "haikuporter --nochroot" is implemented and a user chooses a `TREE_PATH` containing white spaces then the quotes would **still** be unneeded because the shell does not need them when it is doing an assignement which does not include white spaces. The quotes are only required if there are white spaces in the assignement **itself**. If the white spaces are **only** in the variables being refered then the quotes are **not** required.

So, since most recipes currently follow the template we have in the wiki, I suggest that we stick to it.

Reply:
PS, @korli didn't add them, that was my fault :)

Reply:
In 53e790e14e7ad2c6dcc4d97d26cd538fdf3c5035 @korli changed `commandBinDir=$binDir` to `commandBinDir="$binDir"` for:
* [app-arch/zstd/zstd-1.1.2.recipe](https://github.com/haikuports/haikuports/commit/53e790e14e7ad2c6dcc4d97d26cd538fdf3c5035#diff-16d36e0aa549977d29c4ad4a90ad8528R20)
* [app-editors/emacs/emacs-26.1.recipe](https://github.com/haikuports/haikuports/commit/53e790e14e7ad2c6dcc4d97d26cd538fdf3c5035#diff-1ce5c71d39fa3f430738cf5752777e98R27)

I don't remember if there were other similar changes. But I'm quite sure that most recipes are still using a style consitent with the wiki.

Reply:
More of these in b369b0e338abbafc4dd08c38fc4bf048ab559be4:
* [media-libs/alembic/alembic-1.7.8.recipe](https://github.com/haikuports/haikuports/commit/b369b0e338abbafc4dd08c38fc4bf048ab559be4#diff-fdcaf416fea60f28e463587e604f4250R34)
* [media-libs/lcms/lcms-2.9.recipe](https://github.com/haikuports/haikuports/commit/b369b0e338abbafc4dd08c38fc4bf048ab559be4#diff-6690ae642865718f5ad4d786731c80b4R20)

I really don't understand why he made these edits. They were unneeded and, above all, not consistent with the wiki.

Reply:
Good news: we still have over 120 recipes using the style recommended by the wiki, and only 15 recipes using a non consistent style.